src/app/containers/Timestamp/index.test.jsx

```diff
@@ -16,9 +16,54 @@ const regexDatetime = /[0-9]{1,2} \w+ [0-9]{4}[,] [0-9]{2}[:][0-9]{2} \w+/;
 const renderedTimestamps = jsx => render(jsx).get(0).children; // helper as output is wrapped in a grid
```

Comment:
We should update the regex for `regexDate` so that it matches when there is a date only, and not when there is date and time. This will prevent false positives in our unit tests.